flexget/plugins/cli/database.py

```diff
@@ -77,16 +79,21 @@ def reset_plugin(options):
             console('Unable to reset %s: %s' % (plugin, e.message))
 
 
+def backup(manager):
+    fire_event('manager.backup_db', manager)
```

Comment:
So I'll add an argument to the database subparser from the backup pkugin. Got it, although I'm not sure as to why firing that event doesn't feel right to you